rawat-shashank/tbd-components
# Add assets

# Description
Adds Icon component. which add SVG to react component to be used in another project.

## Type

This PR should label the type of changes.


## Testing

## Steps to manually verify the changes:

1. _Exercise_: Run `npm run dev`
2. _Verify_: Ensure http://localhost:6006/?path=/docs/icons-icon--arrow-up  loads with expected content.

# Design

https://www.figma.com/file/coSpIvTKbEnqd4cmo8ktQY/Portfolio-UI-Kit-(Community)?node-id=329%3A3

# Review Checklist

PR should follow conventional commit.

- [x] Branch is update with latest code from master.
- [x] This PR is provided with a description, testing steps, and adequate context for reviewers.
- [x] This PR is label with issue number.
- [x] This PR contains proper documentation(JSDocs) in storybook.
- [ ] Unit tests are added/updated to provide the coverage for PR changes.
